[Haploview](http://www.broadinstitute.org/haploview/haploview). 

[HaploReg v3](http://www.broadinstitute.org/mammals/haploreg/haploreg_v3.php) 
is a full featured genome browsers for developing hypotheses of the non-coding variants on diseases, clinical phenotypes and normal variations. It uses LD information from the 1000 Genomes Project to explore regulatory motif alterations (ENCODE experiments), enhancer annotation (Epigenome Mapping Consortium), eQTLs (GTex eQTL browser), chromatin states and etc.

Using [HaploReg](http://nar.oxfordjournals.org/content/40/D1/D930.full) and RegulomeDB to [mine](http://www.genome.gov/Pages/Research/ENCODE/ASHG_2013_Using_HaploReg_RegulomeDB_to_Mine_ENCODE_Data.pdf) ENCODE data.

[RegulomeDB](http://regulome.stanford.edu/about) is a [database](http://genome.cshlp.org/content/22/9/1790.long) that annotates SNPs with known and predicted regulatory elements in the intergenic regions of the H. sapiens genome.

[SNAP](http://www.broadinstitute.org/mpg/snap/) finds proxy SNPs based on LD, physical distance and/or membership in selected commercial genotyping arrays (the International HapMap Project & the 1000 Genomes Project). 
[SNPinfo Web Server](http://snpinfo.niehs.nih.gov/) for SNP selection and functional information

HaploReg is a full featured genome browsers for developing hypotheses of the non-coding variants on diseases, clinical phenotypes and normal variations. It uses LD information from the 1000 Genomes Project to explore regulatory motif alterations (ENCODE experiments), enhancer annotation (Epigenome Mapping Consortium), eQTLs (GTex eQTL browser), chromatin states and etc.

[Hapmap](http://hapmap.ncbi.nlm.nih.gov/cgi-perl/gbrowse/hapmap28_B36/) is to identify and catalog genetic similarities and differences in human beings.

A Compilation of Some Available [Software](http://www.genes.org.uk/software/LD-software.shtml) for [Linkage Disequilibrium Analysis](https://www.biostars.org/p/2909/).
